Sources/Build/BuildParameters.swift

```diff
@@ -155,6 +164,10 @@ public struct BuildParameters: Encodable {
         return dataPath.appending(components: "..", configuration.dirname + ".yaml")
     }
 
+    public var pifManifest: AbsolutePath {
+        return buildPath.appending(components: "manifest.pif")
```

Comment:
@aciidb0mb3r As you can see, we are storing the PIF manifest in the build path as it is specific to the platform and configuration (because of conditional dependencies).